### DISCUSSION TITLE
complete acl object template

### DISCUSSION THREAD

**Discussion by at1984z** [Edgio/waflz#307] [Q&A]
where can I find a complete template or schema of new acl object?

**Comment by tinselcity** [comment]
The schema is de-facto defined in the `.proto` files [here](https://github.com/Edgio/waflz/blob/master/proto/acl.proto).

There's some public facing product documentation I think which is related (_but could be out of date?_):
- [https://docs.edgecast.com/cdn/Content/Web-Security/Access-Rules.htm](https://docs.edgecast.com/cdn/Content/Web-Security/Access-Rules.htm)

I'm not currently affiliated with the development, maybe @dev0z can relay more current information if they have any.

Thanks and good luck!
